python/rbac/upgrade/role_spec.yaml

```diff
@@ -59,6 +56,737 @@ roles:
+
+    - name: Unmap hosts
+      description: The ability to unmap hosts to the Cluster
+      parametrized_by:
+        - cluster
+      module_name: rbac.roles
+      class_name: ObjectRole
+      init_params:
+        app_name: cm
+        model: Cluster
+      apps:
+        - label: cm
+          models:
+            - name: host
+              codenames:
+                - delete
+            - name: cluster
+              codenames:
+                - change
+
+    - name: Upgrade cluster bundle
+      description: Update the cluster bundle
+      parametrized_by:
+        - cluster
+      module_name: rbac.roles
+      class_name: ObjectRole
+      init_params:
+        app_name: cm
+        model: Cluster
+      apps:
+        - label: cm
+          models:
+            - name: upgrade
+              codenames:
+                - add
+
+    - name: Upgrade provider bundle
+      description: Update the provider bundle
+      parametrized_by:
+        - provider
+      module_name: rbac.roles
+      class_name: ObjectRole
+      init_params:
+        app_name: cm
+        model: HostProvider
+      apps:
+        - label: cm
+          models:
+            - name: upgrade
+              codenames:
+                - add
+
+    - name: Remove cluster
+      description: The ability to remove cluster
+      parametrized_by:
+        - cluster
+      module_name: rbac.roles
+      class_name: ObjectRole
+      init_params:
+        app_name: cm
+        model: Cluster
+      apps:
+        - label: cm
+          models:
+            - name: cluster
+              codenames:
+                - delete
+
+    - name: Remove provider
+      description: The ability to remove provider
+      parametrized_by:
+        - provider
+      module_name: rbac.roles
+      class_name: ObjectRole
+      init_params:
+        app_name: cm
+        model: HostProvider
+      apps:
+        - label: cm
+          models:
+            - name: hostprovider
+              codenames:
+                - delete
+
+    - name: Remove bundle
+      description: The ability to remove bundle
+      module_name: rbac.roles
+      class_name: ModelRole
+      apps:
+        - label: cm
+          models:
+            - name: bundle
+              codenames:
+                - delete
+
+    - name: View ADCM settings
+      description: View purely adcm config
+      module_name: rbac.roles
+      class_name: ObjectRole
+      init_params:
+        app_name: cm
+        model: ADCM
+        filter:
+          name: ADCM
+      apps:
+        - label: cm
+          models:
+            - name: objectconfig
+              codenames:
+                - view
+
+    - name: Compare ADCM settings
+      description: compare settings purely adcm config
+      module_name: rbac.roles
+      class_name: ObjectRole
+      init_params:
+        app_name: cm
+        model: ADCM
+        filter:
+          name: ADCM
+      apps:
+        - label: cm
+          models:
```

Comment:
Is it different from another "compare * settings"? it does not include objectconfig model